src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/CaCaseloadService.kt

```diff
@@ -273,13 +269,7 @@ class CaCaseloadService(
     // Default status (if not overridden below) will show the case as clickable on case lists
     var licenceStatus = NOT_STARTED
 
-    if (isBreachOfTopUpSupervision(c)) {
```

Comment:
removing these conditions as we are filtering out **recall and breachOfTopUpSupervision cases** in the eligibility check which is called before this.